main.go

```diff
@@ -73,6 +73,9 @@ func main() {
 		esExportIndicesMappings = kingpin.Flag("es.indices_mappings",
 			"Export stats for mappings of all indices of the cluster.").
 			Default("false").Bool()
+		esNoExportIndexAliases = kingpin.Flag("es.no-aliases",
+			"Exclude informational alias metrics.").
+			Default("false").Bool()
```

Comment:
This should be flipped around.

```suggestion
		esNoExportIndexAliases = kingpin.Flag("es.aliases",
			"Exclude informational alias metrics.").
			Default("true").Bool()
```

The flag library provides boolean flags like `--no-es.aliases`. The current way would be `--no.es.no-alises`.